browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
+    'no-extra-parens': 0,
+    'no-inline-comments': 0,
+    'react/jsx-uses-react': [ 2, { pragma: 'dom' } ],
+    'space-before-function-paren': [ 2, { anonymous: 'always', named: 'never' } ]
```

Comment:
I'd like to avoid purely stylistic overrides at the `es6`, `node`, etc. level because it'll fragment our style, sorta defeats the purpose of an org-wide style, and this one goes against the style in most of our other projects. Is this something we can override at the app level instead?